README.md

```diff
@@ -67,6 +67,31 @@ If you get the error ``binary `cargo-embed` already exists`` during installation
 
 </details>
 
+<!-- Creating the project -->
+<details open="open">
+  <summary><h2 style="display: inline-block" id="project-creation">Creating your project</h2></summary>
+
+### Using `cargo-generate`
+
+```sh
+cargo generate --git https://github.com/rp-rs/rp2040-project-template
```

Comment:
I'd prefer the brief syntax that assumes github hosting, in case folks are typing this out instead of copy-pasting.

```suggestion
cargo generate rp-rs/rp2040-project-template
```

Reply:
Also we have installation instructions for everything else, but not for cargo generate.